locale/pt_BR/ManageIQ_Providers_Hawkular.po

```diff
@@ -0,0 +1,39 @@
+# renata <translator01_redhat@roundtableinc.net>, 2017. #zanata
+msgid ""
+msgstr ""
+"Project-Id-Version: ManageIQ_Providers_Hawkular 1.0.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-10-31 15:41+0100\n"
+"PO-Revision-Date: 2017-11-08 12:39-0500\n"
+"Last-Translator: renata <translator01_redhat@roundtableinc.net>\n"
+"Language-Team: Portuguese (Brazil)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: pt-BR\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"X-Generator: Zanata 3.9.6\n"
+
+#: ../app/models/manageiq/providers/hawkular/inventory/server_operations.rb:57
+msgid "%{operation} operation for %{server} %{status}"
+msgstr "Operação %{operation} para o %{server} %{status}"
+
+#: ../app/models/manageiq/providers/hawkular/middleware_manager/middleware_diagnostic_report.rb:95
+msgid "Generation of JDR report was requested by a user."
+msgstr "A geração do relatório JDR foi solicitada por um usuário."
+
+#: ../app/models/manageiq/providers/hawkular/middleware_manager/middleware_diagnostic_report.rb:59
+msgid "Reached generation timeout."
+msgstr "Tempo limite de geração atingido."
```

Comment:
Wouldn't this be better as `Tempo limite para geração atendido`?

Reply:
I quite frankly don't know. These translations were made by an external company and
I'm just including whatever they created.